src/Keboola/DbExtractor/Extractor/Redshift.php

```diff
@@ -17,6 +17,8 @@ class Redshift extends Extractor
 {
     private $dbConfig;
 
+    const BATCH_SIZE = 1000;
```

Comment:
chceme treba ho dat do ConfigDefinition?  Ale muzem to udelat potom

Reply:
Zatim bych to nedelal. 1000 je maximum pro single node cluster. A uzivatel, ktery si nastavuje DB spojeni asi bezne nezna jak vykonny ten redshift je.